static/js/formatters-internal.js

```diff
@@ -525,24 +525,30 @@ export function price(fieldValue = {}, locale) {
 }
 
 /**
- * Returns a localized price range string for the given price range ($-$$$$) and country code (ISO format)
+ * Returns a localized price range string for the given price range ($-$$$$) and country code (ISO format).
+ * If country code is invalid or undefined, use locale of the site to determine the currency symbol.
+ * If all else fails, use the default priceRange with dollar sign.
  * @param {string} defaultPriceRange The price range from LiveAPI entity
  * @param {string} countrycode The country code from LiveAPI entity (e.g. profile.address.countryCode)
  * @return {string} The price range with correct currency symbol formatting according to country code
  */
 export function priceRange(defaultPriceRange, countryCode) {
-  if (!defaultPriceRange || !countryCode) {
+  if (!defaultPriceRange) {
     console.warn(`No price range or country code given.`);
```

Comment:
This no longer checks for country code, right? So, it should only warn that no price range is given